hardware/blink1/blink1.html

```diff
@@ -26,8 +25,9 @@
 
 <script type="text/javascript">
     RED.nodes.registerType('blink1',{
-        category: 'output',
```

Comment:
thought we said leave as output (for now at least)
(and input for the blink-in one)

Reply:
In chat you didn't say to leave it at output. I feel having "blink1 out" in category _output_ and "blink1 in" in _input_ makes it hard to find things. Right now I pretty much always have one category per "device" (e.g. Homeassistant = one category, Philips Hue = one category) and I feel blink1 should also be a category if more than one nodes exist. I don't have any other devices in "input" nor in "output".

But again this is opinionated. Go ahead and change it it doesn't hurt functionality.